modules/guestbook/guestbook-service/src/main/java/br/com/objective/training/search/EntryModelDocumentContributor.java

```
@@ -46,37 +48,39 @@ public void contribute(Document document, Entry entry) {
-
         try {
-            defaultLocale = PortalUtil.getSiteDefaultLocale(entry.getGroupId());
-        } catch (PortalException e) {
-            throw new Exception(e);
-        }
+            Locale defaultLocale = PortalUtil.getSiteDefaultLocale(entry.getGroupId());
 
-        document.addDate(Field.MODIFIED_DATE, entry.getModifiedDate());
-        document.addText("email", entry.getEmail());
+            document.addDate(Field.MODIFIED_DATE, entry.getModifiedDate());
+            document.addText("email", entry.getEmail());
 
-        String localizedTitle = LocalizationUtil.getLocalizedName(Field.TITLE, defaultLocale.toString());
-        String localizedMessage = LocalizationUtil.getLocalizedName(Field.CONTENT, defaultLocale.toString());
+            String localizedTitle = LocalizationUtil.getLocalizedName(Field.TITLE, defaultLocale.toString());
+            String localizedMessage = LocalizationUtil.getLocalizedName(Field.CONTENT, defaultLocale.toString());
 
-        document.addText(localizedTitle, entry.getName());
-        document.addText(localizedMessage, entry.getMessage());
+            document.addText(localizedTitle, entry.getName());
+            document.addText(localizedMessage, entry.getMessage());
 
-        long guestbookId = entry.getGuestbookId();
+            long guestbookId = entry.getGuestbookId();
 
-        Guestbook guestbook = _guestbookLocalService.getGuestbook(guestbookId);
+            Guestbook guestbook = _guestbookLocalService.getGuestbook(guestbookId);
 
-        String guestbookName = guestbook.getName();
+            String guestbookName = guestbook.getName();
 
-        String localizedGbName = LocalizationUtil.getLocalizedName("guestbookName", defaultLocale.toString());
+            String localizedGbName = LocalizationUtil.getLocalizedName("guestbookName", defaultLocale.toString());
+
+            document.addText(localizedGbName, guestbookName);
+        } catch (PortalException pe) {
```

Should we throw this exception?